test/quantization/test_quantize_fx.py

```diff
@@ -2881,6 +2882,7 @@ def _test_model_impl(
             qeager.eval()
 
             # print('ref after quantization:', qeager)
+            qeager_input = copy.deepcopy(input_value)
```

Comment:
Is this used anywhere else?

Reply:
no, will remove, it's added when debugging